src/assets/v1.3.0/proxy_factory.json

```diff
@@ -1,9 +1,16 @@
 {
   "defaultAddress": "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
-  "released": false,
+  "released": true,
   "contractName": "ProxyFactory",
```

Comment:
Shouldn't be `GnosisSafeProxyFactory`?

Reply:
It actually doesn't matter, we named it GnosisSafeProxFactory to avoid naming conflicts when using the contract files.

Reply:
I can update it later